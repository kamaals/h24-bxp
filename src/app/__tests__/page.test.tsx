import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import HomePage from "../page";

describe("RootLayout", () => {
  it("renders the RootLayout component", () => {
    const result = render(<HomePage />);
    expect(result).toMatchSnapshot();
  });
});
