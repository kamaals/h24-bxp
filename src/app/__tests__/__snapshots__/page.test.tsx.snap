// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`RootLayout > renders the RootLayout component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section>
        <h1>
          Welcome!
        </h1>
        <header>
          <a
            href="/dashboard"
          >
            Dash
          </a>
          <a
            href="/login"
          >
            Login
          </a>
        </header>
        <main>
          <p>
            Hello and welcome!
          </p>
        </main>
        <footer>
          Hi
        </footer>
      </section>
    </div>
  </body>,
  "container": <div>
    <section>
      <h1>
        Welcome!
      </h1>
      <header>
        <a
          href="/dashboard"
        >
          Dash
        </a>
        <a
          href="/login"
        >
          Login
        </a>
      </header>
      <main>
        <p>
          Hello and welcome!
        </p>
      </main>
      <footer>
        Hi
      </footer>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
